resources/views/tasks/show.blade.php

```diff
@@ -1,12 +1,102 @@
 @extends('layouts.layout')
 
 @section('content')
-<div class="container" id="task">
-    <task-view
-        process-uid="{{$process->uid}}" instance-uid="{{$instance->uid}}" token-uid="{{$token->uid}}" form-uid="{{$token->definition['formRef']}}"
-        :data="{{json_encode($data)}}" token-created="{{$token->delegate_date}}" token-completed="{{$token->finish_date}}"
-        user-uid="{{$token->user->uid}}" user-name="{{$token->user->fullname}}" user-avatar="{{$token->user->avatar}}"></task-view>
-</div>
+<!-- <nav>
+    <ul class="nav row" style="height:40px; background-color:#b6bfc6;">
+      <li class="nav-item col-3 align-self-center">
+        <a class="nav-link active text-light p-0 ml-3" href="#"><i class="fas fa-long-arrow-alt-left fa-lg mr-2"></i>BACK TO REQUEST DETAILS</a>
+      </li>
+      <li class="nav-item col-3 align-self-center text-right">
+        <a class="nav-link text-light p-0" href="#"><h4 class="m-0">Task: Approve Leave of Absence</h4></a>
+      </li>
+      <li class="nav-item col align-self-center">
+        <a class="nav-link p-0" href="#" ><span class="pill badge-light p-1 pl-2 pr-2" style="border-radius: 20px;"><i class="fas fa-circle text-primary mr-2"></i><span>Pending</span></a>
+      </li>
+    </ul>
+</nav> -->
+    <div class="d-flex container mt-3">
+        <div class="col-9 mt-5 mb-5">
+            <div class="container mb-5 p-5">
+                <h3 class="font-weight-bold">Request</h3>
+                <form>
+                    <div class="form-group">
+                        <label for="groupName">Start Date</label>
+                        <input type="text" class="form-control" id="groupNameStart">
+                    </div>
+                    <div class="form-group">
+                        <label for="groupName">End Date</label>
+                        <input type="text" class="form-control" id="groupNameEnd">
+                    </div>
+                    <div class="form-group mb-3">
+                        <label for="description">Reason</label>
+                        <textarea class="form-control" id="description" rows="3"></textarea>
+                    </div>
+                    <h3 class="font-weight-bold mb-3">Approve?</h3>
```

Comment:
The h3 should already have these classes as part of the tag. We shouldn't override the classes - a H3 should always look like all of the other H3s